### DISCUSSION TITLE
Referencing current publication number in eForms change/modification notice

### DISCUSSION THREAD

**Discussion by EEanalyst** [OP-TED/eForms-SDK#314] [Q&A] [answered]
When creating a change or modification notice, we need to reference the previous notice. For that we should use BT-758 or BT-1501.
On Identifiers and references page it says that if we want to reference the TEDXML notice, we should use ojs-notice-id and the format is like this: XXXXXXXXXX-YYYY
But currently we have publication IDs in this format 2022/S 001-001234. To add the reference to the eForms change/modification notice, should we format the publication number to be like this: 00001234-2022?
Why can't we reference the no doc ojs as it is currently?

**Comment by YvesJo** [comment, marked as answer]
For "TED search", the two formats are acceptable Notice Publication Numbers. 
![image](https://user-images.githubusercontent.com/42802091/214530591-bc608847-8acf-41ad-bca1-0204984fe5e4.png)

There is a need to refer to a notice, not to the OJ publication that contains it.
On TED, notices are identified using the Document number.
![image](https://user-images.githubusercontent.com/42802091/214530334-004b6e86-6a22-4a87-aabd-7c9fe238d9cd.png)

The Document Number is less prone to error (absence of white spaces, reduced number of characters ...)

Also for business related questions, please submit them to the Helpdesk: ted@publications.europa.eu

**Reply by EEanalyst** [reply]
To clarify, in old XML the document number field is NO_DOC_EXT?
For example:
```
<TED_ESENDERS VERSION="R2.0.9.S01" 
xmlns="[http://formex.publications.europa.eu/ted/schema/reception/R2.0.9.S01" 
<SENDER>    
<IDENTIFICATION>     
  ...       
<NO_DOC_EXT>2016-007368</NO_DOC_EXT>    
</IDENTIFICATION>  
  ..  
</SENDER> 
 ..
</TED_ESENDERS>
```
NO_DOC_EXT is in the format of YYYY-XXXXXX but according to documentation it should be like this: 
![image](https://user-images.githubusercontent.com/121294182/214827668-3d7457f3-7857-4aff-85b7-3dfbab0aa58a.png)
Is the NO_DOC_EXT element value correct to use in change or modification notice XML to identify the previous notice we are changing? And if it is, should we modify it to fit the eForms  XXXXXXXX-YYYY requirement?
